modules/ROOT/pages/queries/concepts.adoc

```diff
@@ -0,0 +1,82 @@
+:description: this page describes the three fundamental concepts of a Cypher query: nodes, relationships, and patterns.
+
+[[core-concepts]]
+= Core concepts
+
+Fundamentally, a Neo4j graph database consists of three core entities: nodes, relationships, and patterns.
```

Comment:
I believe that is correct - but I will double check :)

Reply:
I don't have a formal definition of entity, but I tend to think of them as of the _type of objects_ a query can return: values/properties, nodes, relationships, patterns. Another way for me to think about that would be the things I can assign a variable to. Does this match your intuition/definition @gem-neo4j ?

Reply:
Well technically paths are returned, not patterns, and that has been my main hang up :)

Reply:
It was an awkward conversation I had with Jens where we talked about paths/patterns, until I realized that in my mind I had read `paths` in that section all along :facepalm: so yeah, paths it will be.